indexing-hadoop/src/main/java/io/druid/indexer/Utils.java

```diff
@@ -45,6 +46,17 @@
   public static OutputStream makePathAndOutputStream(JobContext job, Path outputPath, boolean deleteExisting)
       throws IOException
   {
+    return makePathAndOutputStream(job, outputPath, deleteExisting, -1);
```

Comment:
can we have two separate methods makePathAndOutputStream and makePathAndBufferedOutputStream that takes buffer size  ?